### ISSUE TITLE
Experiment indications scripts are loaded twice

### ISSUE THREAD

**Issue by rigwild** [prise-3d/Thesis-WebExpe-Django#3]
Experiment indications scripts are loaded twice, which leads to front's JavaScript to output the following error:
```
indications.js:1 Uncaught SyntaxError: Identifier 'KEYCODE_Q' has already been declared
```

The followings files are responsible for it.

https://github.com/prise-3d/Thesis-WebExpe-Django/blob/1c54133c39b1597fcbf088875e677a7c538a2a73/expe/templates/expe/expe_indications.html#L17-L19

https://github.com/prise-3d/Thesis-WebExpe-Django/blob/1c54133c39b1597fcbf088875e677a7c538a2a73/expe/templates/base.html#L54-L59

![image](https://user-images.githubusercontent.com/26366184/65819009-bcbae180-e217-11e9-85b6-d6a03b8f26c3.png)

**Comment by rigwild** [comment]
Happens at `/admin/results` too (probably the same cause).
![image](https://user-images.githubusercontent.com/26366184/65819222-2a680d00-e21a-11e9-91e8-e23f3893e13f.png)

**Comment by jbuisine** [comment]
Due to `{% block javascripts %}` which is missplaced into templates and called twiced.

Fixed into [commit](https://github.com/prise-3d/Thesis-WebExpe-Django/commit/101f2c9bdde27a3d7410608663eaa560caaf1a32) and available in release v0.1.6.

Thanks for your report!
